### ISSUE TITLE
Nodemon fails to restart the app

### ISSUE THREAD

**Issue by prathamesh0007** [remy/nodemon#1854]
**Please help me out I'm stuck here since two days. Failed to find solution all over internet.**
- Operating system/terminal environment: **Win 10/Hyper+Gitbash**
- Using Docker? What image: **Not using Docker**
- Command you ran: **npm install -g nodemon**

### Expected behaviour
I expected it to restart my App

### Actual behaviour
It starts my app smoothly but on change fails to restart the App

### Steps to reproduce
I refered to Angela Yu's Udemy course(Outdated) and I installed following things in the given sequence gitbash->node.js -> nodemon
![image](https://user-images.githubusercontent.com/63106675/117585085-2b6b5b00-b12e-11eb-8810-14d3bc97d5c4.png)

**Comment by 0neMiss** [comment]
I am also having this issue. 

nodemon version 2.07
node version 12.16.3
windows 10

package.json

```
{
  "name": "new",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {

    "server": "set NODE_ENV=development&&nodemon index.js",
    "client": "npm run start --prefix client",
    "dev": "concurrently \"npm run server\" \"npm run client\"",
    "start": "node index.js",
    "heroku-postbuild": "NPM_CONFIG_PRODUCTION=false npm install --prefix client && npm run build --prefix client"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "aws-sdk": "^2.886.0",
    "bcrypt": "^5.0.1",
    "dotenv": "^8.2.0",
    "express": "^4.17.1",
    "express-session": "^1.17.1",
    "jsonwebtoken": "^8.5.1",
    "knex": "^0.95.4",
    "knexnest": "^1.0.0",
    "moment": "^2.29.1",
    "moment-timezone": "^0.5.33",
    "mongoose": "^5.12.2",
    "morgan": "^1.10.0",
    "multer": "^1.4.2",
    "passport": "^0.4.1",
    "passport-facebook": "^3.0.0",
    "passport-google-oauth20": "^2.0.0",
    "passport-jwt": "^4.0.0",
    "passport-linkedin-oauth2": "^2.0.0",
    "passport-local": "^1.0.0",
    "path": "^0.12.7",
    "pg": "^8.6.0",
    "stripe": "^8.140.0",
    "uuid": "^8.3.2"
  },
  "devDependencies": {
    "concurrently": "^6.0.0",
    "nodemon": "^2.0.7"
  }
}
```
![image](https://user-images.githubusercontent.com/56180519/119174213-4eebab00-ba36-11eb-80c0-7516fa879586.png)

I was advised it could be a path issue. How would I go about troubleshooting that and fixing it? 
I'm working with a team who use the exact same codebase as me who all have no issues with nodemon currently.

**Comment by 0neMiss** [comment]
I also found that for me at least this seems to be specific to npm. The other day I started a project using yarn and nodemon worked fine there. Nodemon is also tracking my changes to some degree, and if there is a fatal error that wont compile then it will actually restart when its resolved.

**Comment by hsangrento** [comment]
Same issue,
Windows 10 2004, 19041.985
"nodemon": "^2.0.7",
Node.js v14.17.0.

**Comment by joshuasorkin** [comment]
Same issue.
Windows 10, Version 10.0.19402.1052
nodemon 2.0.7
Node.js 14.16.1

**Comment by remy** [comment]
Windows has had a fix deployed in nodemon@2.0.9 for restarting - this should help. I'll reopen if the OP still has the problem.
